docker/Dockerfile

```diff
@@ -0,0 +1,70 @@
+#
+# Docker layer for building Gramine
+# from https://github.com/gramineproject/gramine/blob/master/packaging/docker/Dockerfile
+#
+
+ARG UBUNTU_IMAGE=ubuntu:22.04
+
+FROM ${UBUNTU_IMAGE}
+
+ARG UBUNTU_CODENAME=jammy
+WORKDIR /usr/src/raiko
```

Comment:
I think you can't place raiko in `/usr` https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/usr.html
`/opt/raiko` maybe be a good place https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/opt.html